From: Roel van Meer <roel@1afa.com>
Date: Sun, 23 Apr 2017 13:04:10 +0100
Subject: Set absolute path to z-push in gab2contacts

Upstream does this with sed in the rules file, but using a patch is
cleaner imho.
---
 tools/gab2contacts/gab2contacts.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/gab2contacts/gab2contacts.php b/tools/gab2contacts/gab2contacts.php
index 582734d..7f2e4ac 100755
--- a/tools/gab2contacts/gab2contacts.php
+++ b/tools/gab2contacts/gab2contacts.php
@@ -28,7 +28,7 @@
 // The path set by default is as required for a GIT checkout.
 // It is also possible to use absolute path to z-push installation, e.g.
 // /usr/share/z-push/
-define('PATH_TO_ZPUSH', '../../src/');
+define('PATH_TO_ZPUSH', '/usr/share/z-push/');
 
 /************************************************
  * MAIN
@@ -240,4 +240,4 @@ class GAB2ContactsCLI {
     static public function GetWorker() {
         return self::$contactWorker;
     }
-}
\ No newline at end of file
+}
